FFXCutsceneRemover/Resources/Transitions.cs

```diff
@@ -228,19 +228,14 @@ static class Transitions
             // END OF MOONFLOW
             // START OF GUADOSALAM
             { new GameState { RoomNumber = 135, Storyline = 1085 }, new Transition { RoomNumber = 135, Storyline = 1096, SpawnPoint = 0, Description = "Meet Tromell + Customise tutorial"} },
-            { new GameState { RoomNumber = 163, Storyline = 1096, State = 1}, new Transition { RoomNumber = 163, Storyline = 1104, SpawnPoint = 1, Description = "Tromell invites the gang in"} },           
```

Comment:
Decided to skip all the movement in Seymour's house. Far easier to implement, and we're not missing out on key chests or anything from it, nor should it affect muscle memory for the speedrunners